sbt_common/elasticsearch/src/test/scala/uk/ac/wellcome/elasticsearch/test/fixtures/ElasticsearchFixtures.scala

```diff
@@ -60,17 +60,9 @@ trait ElasticsearchFixtures
 
   // Elasticsearch takes a while to start up so check that it actually started
   // before running tests.
-  //
-  // AWLC: I have commented this out because it was throwing an inexplicable
-  // NullPointerException in the ingestor feature test, and I'd exhausted my
-  // patience trying to work out why.  Feel free to put it back if you can solve
-  // this particular mystery!
-  //
-  // This was the relevant patch: https://github.com/wellcometrust/platform/pull/3025
-  //
-  // eventually {
-  //   elasticClient.execute(clusterHealth()).await.numberOfNodes shouldBe 1
-  // }
+  eventually {
```

Comment:
I am confuse.

Did @alexwlchan add the comment with the correct code in, but not the code?

Reply:
I think @alexwlchan commented a block of code and added a comment explaining why he did it:  [here](https://github.com/wellcometrust/platform/commit/51b550c5fa3eae6db05529083e51d4c60b4327dd#diff-eb3800e5d2dab8290d7424fd4dc8b1d4R61)